specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,2322 @@
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "Bit",
+            "Kilobit",
+            "Megabit",
+            "Gigabit",
+            "Terabit",
+            "Petabit",
+            "Byte",
+            "Kilobyte",
+            "Megabyte",
+            "Gigabyte",
+            "Terabyte",
+            "Petabyte"
+          ],
+          "x-ms-enum": {
+            "name": "InformationUnit",
+            "modelAsString": true
+          },
+          "description": "The information (data) Unit of measurement."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "TemperatureResolution": {
+      "description": "Represents the temperature entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Temperature",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "Fahrenheit",
+            "Kelvin",
+            "Rankine",
+            "Celsius"
+          ],
+          "x-ms-enum": {
+            "name": "TemperatureUnit",
+            "modelAsString": true
+          },
+          "description": "The temperature Unit of measurement."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "WeightResolution": {
+      "description": "Represents the weight entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Weight",
+      "properties": {
+        "unit": {
+          "type": "string",
+          "enum": [
+            "Unspecified",
+            "Kilogram",
+            "Gram",
+            "Milligram",
+            "Gallon",
+            "MetricTon",
+            "Ton",
+            "Pound",
+            "Ounce",
+            "Grain",
+            "PennyWeight",
+            "LongTonBritish",
+            "ShortTonUS",
+            "ShortHundredWeightUS",
+            "Stone",
+            "Dram"
+          ],
+          "x-ms-enum": {
+            "name": "WeightUnit",
+            "modelAsString": true
+          },
+          "description": "The weight Unit of measurement."
+        }
+      },
+      "required": [
+        "unit"
+      ]
+    },
+    "CurrencyResolution": {
+      "description": "Represents the currency entity resolution model.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseResolution"
+        },
+        {
+          "$ref": "#/definitions/QuantityResolution"
+        }
+      ],
+      "type": "object",
+      "x-ms-discriminator-value": "Currency",
+      "properties": {
+        "iso4217": {
```

Comment:
@nourhaanfarid, @aurghob pointed out you changed in this in XX, but "ISO4217" already GA'd at least in 2022-05-01 and can't be changed now. Besides, it's also not updated in the /dev folder...not that it should be. These documents cannot typically make modifications apart from additions or changes to docs.

Reply:
@heaths if my memory serves me right, the issue was a validation error in the pipeline but as you correctly pointed out this is already GA'd. Can we keep it as "ISO4217" and deal with the validation error instead?

Reply:
You should not change an enum value in a GA API just because some validation tool complains about it. The number one rule is do not break customers -- meaning do not make incompatible changes to a GA API. Just live with the validation error or find a way to suppress it.

Reply:
Thanks I'm reverting the relevant commit for this change based on this conversation and an offline conversation with @fxs130430

Reply:
Reverted